common/libs/nodes/rpc/types.js

```diff
@@ -1,8 +1,9 @@
-// @flow
+// don't use flow temporarily
```

Comment:
@bfillmer There were a large amount of flow errors here that may be irrelevant after your work in https://github.com/MyEtherWallet/MyEtherWallet/pull/122 is complete, so I've temporarily disabled flow for this file so as not to have to trudge through that in the case that it is indeed irrelevant.